packages/css/src/grid/page-grid.scss

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../utils/breakpoint";
+
+.amsterdam-page-grid {
+  column-gap: 2%;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  max-width: 1440px;
```

Comment:
Should we extract to a constant, like the breakpoint? If so, which of the other values here as well?

Reply:
I’ve extracted tokens for `column-gap`, `max-width`, and `column-count`.